core/src/apps/base.py

```diff
@@ -1,25 +1,24 @@
-from typing import TYPE_CHECKING
-
 import storage.cache as storage_cache
 import storage.device as storage_device
 from trezor import config, utils, wire, workflow
 from trezor.enums import HomescreenFormat, MessageType
 from trezor.messages import Success, UnlockPath
+from typing import TYPE_CHECKING
```

Comment:
this change should not be here

Reply:
> There are still unwanted changes.
> 
> As a general rule, we do not want any changes in top-level imports that were sorted "correctly" with the previous version. Reordering is allowed (a) in functions, (b) in "if type_checking" and other conditional blocks, and (c) in try blocks -- because those weren't managed by isort previously, so there might be mistakes.
> 
> I recommend going through https://pycqa.github.io/isort/docs/upgrade_guides/5.0.0.html because it appears that some configuration options are not working as they used to.

Understood. I found the error. 
-[Changes](https://pycqa.github.io/isort/docs/upgrade_guides/5.0.0.html#:~:text=isort%20settings%20no%20longer%20merge%20together%2C%20instead%20they%20override.) 
- [Solution
](https://pycqa.github.io/isort/docs/upgrade_guides/5.0.0.html#:~:text=you%20can%20still%20get%20this%20behavior%20by%20moving%20your%20extra%20modules%20from%20the%20known_standard_library%20setting%20to%20extra_standard_library.)